### ISSUE TITLE
Ordered list items with parentheses don't expand in PDF export

### ISSUE THREAD

**Issue by sappjw** [joemccann/dillinger#861]
If you have a document written like this:
```
1) list item 1
1) list item 2
1) list item 3
```
the Preview pane shows it as
```
1. list item 1
2. list item 2
3. list item 3
```
but the PDF export shows all the numbered list items as "1".

If formatted like this:
```
1. list item 1
1. list item 2
1. list item 3
```
both the Preview pane and PDF export format the numbers the same (as "1", "2", "3", as expected).

**Comment by Shadowclash10** [comment]
i don't believe there are any active contributors. the last commit was 8 months ago, and last release was almost a decade ago lmao
